mmv1/third_party/terraform/services/container/resource_container_cluster_test.go.erb

```diff
@@ -9735,3 +9791,415 @@ resource "google_container_cluster" "primary" {
 }
 `, projectID, randomSuffix, clusterName, networkName, subnetworkName)
 }
+
+func testAccContainerCluster_withAutopilotResourceManagerTags(projectID, clusterName, networkName, subnetworkName, randomSuffix string) string {
+	return fmt.Sprintf(`
+data "google_project" "project" {
+  project_id = "%[1]s"
+}
+
+resource "google_project_iam_member" "tagHoldAdmin" {
+  project = "%[1]s"
+  role    = "roles/resourcemanager.tagHoldAdmin"
+  members = [
+   "serviceAccount:service-${data.google_project.project.number}@container-engine-robot.iam.gserviceaccount.com",
+  ]
```

Comment:
```suggestion
  member = "serviceAccount:service-${data.google_project.project.number}@container-engine-robot.iam.gserviceaccount.com"
```

IAM membership resources manage only one membership in a binding, so cannot take a list as an argument here.